examples/data-objects/table-view/src/grid.ts

```diff
@@ -3,47 +3,30 @@
  * Licensed under the MIT License.
  */
 
-import { KeyCode, Scheduler, Template } from "@fluid-example/flow-util-lib";
 import { colIndexToName } from "@fluid-example/table-document";
 import { SharedMatrix } from "@fluidframework/matrix";
 import { ISheetlet, createSheetletProducer } from "@tiny-calc/micro";
 import { BorderRect } from "./borderstyle";
 import * as styles from "./index.css";
 import { TableView } from "./tableview";
 
-const tableTemplate = new Template({
-    tag: "table",
-    props: { className: styles.view, tabIndex: 0, sortable: true },
-    children: [
-        {
-            tag: "caption",
-            children: [{ tag: "span", props: { textContent: "Table" } }],
-        },
-        {
-            tag: "thead",
-            children: [{
-                tag: "tr",
-                ref: "cols",
-            }],
-        },
-        {
-            tag: "tbody",
-            ref: "body",
-        },
-    ],
-});
-
-const rowTemplate = new Template({ tag: "tr" });
-const headerTemplate = new Template({ tag: "th" });
-const cellTemplate = new Template({ tag: "td" });
-const cellInputTemplate = new Template({ tag: "input", props: { className: styles.inputBox } });
-
 // eslint-disable-next-line unicorn/no-unsafe-regex
 const numberExp = /^[+-]?\d*\.?\d+(?:[Ee][+-]?\d+)?$/;
+
+const enum KeyCode {
```

Comment:
looks like this should continue to be imported from `Keycode`?

Reply:
I elected to split it out with some slight duplication.  There are further other packages (search-menu, client-ui-lib) which also define their own `KeyCode` enums already, and I judged that the benefit of package reduction outweighed the duplication.